src/adapter/z-stack/adapter/endpoints.ts

```diff
@@ -35,10 +35,10 @@ export const Endpoints = [
         appprofid: 0x0104,
         appdeviceid: 0x0400,
         appnumoutclusters: 2,
-        appoutclusterlist: [Zcl.Utils.getCluster('ssIasZone').ID, Zcl.Utils.getCluster('ssIasWd').ID],
+        appoutclusterlist: [Zcl.Cluster.ssIasZone.ID, Zcl.Cluster.ssIasWd.ID],
```

Comment:
Import just `Cluster` here, instead of the whole ZCL spec?

`import Cluster from '../../../zcl/definition/cluster';`